transcriptic/cli.py

```diff
@@ -615,12 +615,19 @@ def select_org_cmd(ctx, organization=None):
     type=click.Path(exists=True, file_okay=True, dir_okay=False),
     help="Path to RSA key used for signing requests",
 )
+@click.option(
+    "--web-staging",
+    is_flag=True,
+    required=False,
+    help="login to staging environment for testing changes in master of web that are not yet released to production",
+)
 @click.pass_context
-def login_cmd(ctx, api_root=None, analytics=True, rsa_key=None):
+def login_cmd(ctx, api_root=None, analytics=True, rsa_key=None, web_staging=False):
     """Authenticate to your Transcriptic account."""
+    ctx.obj.api = Connection()
```

Comment:
is this new line necessary?

Reply:
If you don't overwrite it, it will try to pull the info from an existing ~/.transcriptic config file. This causes issues when switching back and forth between staging and production because the organization a user was on previously in production may not be the 'transcriptic' org on staging. This line fixes that so you don't have to manually change your ~/.transcriptic file to point to an existing org that you belong to.